frontend/src/pages/projects/screens/detail/notebooks/__tests__/useNotebookImageData.spec.ts

```diff
@@ -106,4 +106,21 @@ describe('getNotebookImageData', () => {
     const result = getNotebookImageData(notebook, images);
     expect(result?.imageAvailability).toBe(NotebookImageAvailability.DELETED);
   });
+
+  it('should fail without fix', () => {
```

Comment:
Maybe something like this would be more descriptive:
```suggestion
  it('should fail when custom image shows unexpected Deleted flag', () => {
```